walletlibrary/src/main/java/com/microsoft/walletlibrary/VerifiedIdClient.kt

```diff
@@ -28,6 +29,7 @@ class VerifiedIdClient(
     // Creates an issuance or presentation request based on the provided input.
     suspend fun createRequest(verifiedIdRequestInput: VerifiedIdRequestInput): Result<VerifiedIdRequest<*>> {
         return try {
+            VerifiableCredentialSdk.correlationVectorService.startNewFlowAndSave()
```

Comment:
Does the correlationVectorService increment the number in the networking layer?

Reply:
yes it does.